editor/src/benchmarks.benchmark.ts

```diff
@@ -5,9 +5,9 @@ import { benchmarkOptics } from './core/shared/optics.benchmark'
 import { benchmarkPropertyPathFunction } from './core/shared/property-path.benchmark'
 import { benchmarkGetUniqueUids } from './core/model/get-unique-ids.benchmark'
 
-// await benchmarkBuildTree()
+await benchmarkBuildTree()
```

Comment:
Did you intend to commit this? Or was this just you running the benchmark locally?

Reply:
It was from me running it locally, not sure if there's a good answer to not changing this all the time.